### PR TITLE
Halves handcuff and buckling escape times.

### PR DESCRIPTION
**Pull request by Higgin**
What it says on the tin - this has been getting obnoxious for how many years now? Tighten up the window within which your hostage-taker will have to babysit you for more opportunities to escape.

### CODE REVIEW

**Comment by Sparrowbird611** [comment]
Ok so, I'm gonna have to disagree on the unbuckling, but everything else I agree on.

File: code/game/objects/items/weapons/handcuffs.dm [hunk 1]
```diff
@@ -137,7 +137,7 @@ var/last_chew = 0
 	name = "cable restraints"
 	desc = "Looks like some cables tied together. Could be used to tie something up."
 	icon_state = "cuff_white"
-	breakouttime = 300 //Deciseconds = 30s
+	breakouttime = 400 //Deciseconds = 30s
```
Comment (Sparrowbird611) [inline code/game/objects/items/weapons/handcuffs.dm]: Also, you ought to change this to say 40seconds now.

**Comment by Roland410** [comment]
Getting bucklecuffed means you need to spend 4 minutes to get out. Moving, or being moved resets it. Having to spend 2 minutes in one place (or at least 1 minute for the buckle) while you can't even open public doors etc. has always been an awful design.

**Comment by Sparrowbird611** [comment]
Eh.... I still don't think it ought to go below a minute and thirty seconds. I really would prefer not having to shoot those captives who just spam struggle in their chairs while I'm gloating at them.

**Pull request merged**